modules/apps/commerce/commerce-product-asset-categories-web/src/main/java/com/liferay/commerce/product/asset/categories/web/internal/constants/CommerceProductAssetScreenNavigationConstants.java

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
@daiazius, please rename this file to `CPAssetCategoriesScreenNavigationConstants`. Thx